handbook/ce/customer-support-onboarding.md

```diff
@@ -1,5 +1,204 @@
+* You have signed-up for which playbook(s) you will help with and have a plan for how to approach it
+* You’ve selected your focus blocks and updated our team schedule
+
+### In order to ensure the above declarations are true, you will complete the following tasks:
+
+* Keep helping customers!
+* Schedule a chat with each member of the [Distribution team](https://about.sourcegraph.com/handbook/engineering/distribution). This is the engineering team that we cross paths with the most (they own customer deployment, monitoring, and upgrades).
+* Schedule a chat with each engineering team manager. You can find who they are by starting on the [org chart page](https://about.sourcegraph.com/company/team/org_chart), navigating to each engineering team page, and then looking at the members listed somewhere there. 
+* Sign-up for which playbook(s) you will help develop and have a plan for how to approach it. Note: more on this soon.
+* Select your focus blocks and update our [team schedule](https://about.sourcegraph.com/handbook/ce/support-schedule).
+
+
+
+## By the end of your second month, the following will be true...
+* You have closed at least 25 cases
+* You feel super confident troubleshooting and working with the product
+* You are finding you need help less and less to get started helping a customer
+* You have had a chat with each engineering manager and each member of the Distribution team
+* You have completed the playbook(s) you committed to do
+* You have added or substantially updated at least one section to our official product documentation
+
+### In order to ensure the above declarations are true, you will complete the following tasks:
+
+* Keep helping customers!
+* Follow through on your plan to complete the playbook(s) you committed to do Note: more on this soon.
+* Get ready to contribute to product documentation by following [these steps](https://about.sourcegraph.com/handbook/product/onboarding#get-ready-to-contribute-to-the-docs) (though hold-off on purchasing anything to make gifs until we know we have a need for that)
+* Add or substantially update at least one section to our official product documentation
+
+
+
+## By the end of your third month, the following will be true…
+* You have closed at least 50 cases
+* You find that you rarely need help to get started helping a customer
+* You have continued to update product documentation and the handbook as needed
+* You have identified and made progress on a project during your focus blocks
+* You have completed a career roadmap with your manager
+
+### In order to ensure the above declarations are true, you will complete the following tasks:
+
+* Keep helping customers!
+* Update product documentation and/or the handbook as needed
+* Determine a project to work on, develop a plan, and start working on it during your focus blocks
+* Schedule time with your manager to talk about your career roadmap
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+Common tools used for troubleshooting
```

Comment:
I'm assuming this is just sort of a note to self?

Reply:
Ooops -- forgot to remove it when I was formatting -- yeah notes to self and removed!